pkg/acs/calacs/lib/multk2d.c

```diff
@@ -13,25 +18,57 @@ float k          i: multiply a by this constant
 
 	extern int status;
 
-	int i, j;
-    int dimx, dimy;
-
 	if (k == 1.)
 	    return (status);
 
 	/* science data */
-    dimx = a->sci.data.nx;
-    dimy = a->sci.data.ny;
+    int dimx = a->sci.data.nx;
+    int dimy = a->sci.data.ny;
     
-	for (j = 0;  j < dimy;  j++)
-	    for (i = 0;  i < dimx;  i++)
-		Pix (a->sci.data, i, j) = k * Pix (a->sci.data, i, j);
+    int i, j;
```

Comment:
I would have scoped ``i`` & ``j`` and these should be ``unsigned``. Really I would do:
```c
for (unsigned int j = 0; j < dimy; ++j)
{
...
```
However, this is C99 which we have not yet got to (_c.f._ #88) so
```c
{unsigned j;
for (j = 0; ...}
{
...
}}
```
 Also, just small poke, ``++i`` is more performant than ``i++``, even though pretty much all compilers will do the conversion for you.

Reply:
Also, it looks like you have an auto formatter set up, as there seem to be unnecessary indent changes that will pollute the blame history.

Reply:
Made unsigned ints.  Wanted the C99 version - boo hoo.
No autoformatter, but code whacked out previously.  Regrettable regarding the spaces for blame, but "a woman's gotta do what a woman's gotta do".  At least use "?w=1" at end of URL or "diff settings" to view nicely.